provisioning/Build.ps1

```diff
@@ -49,15 +49,18 @@ try {
     Write-OutPut "Npm.ps1 FAILED"
     exit 1
 }
-#try {
-#    $build = Join-Path $originalBuildScriptPath "Build-Windows.ps1"
-#    Write-OutPut "Running windows script: $build"
-#    iwr "$windowsBootstrapURL/Build.ps1" -UseBasicParsing -OutFile $build
-#    Invoke-Expression $build -skipNpm
-#} catch {
-#    Write-OutPut "Build.ps1 FAILED"
-#    exit 1
-#}
 
 $npm = "npm" -f $env:SystemDrive
 Invoke-Command $npm "install" $mainDir
+
+try {
+    $tests = Join-Path $originalBuildScriptPath "Tests.ps1"
+    $fullPath = Join-Path $originalBuildScriptPath "../node_modules/gpii-windows/provisioning/"
+    $args = "-originalBuildScriptPath $fullPath"
+    Write-OutPut "Running windows script: $tests"
+    iwr "$windowsBootstrapURL/Tests.ps1" -UseBasicParsing -OutFile $tests
```

Comment:
is there a reason for fetching these remotely, instead of using the ones from our local gpii-windows repository (that we've fetched in line 53)?